src/main/java/com/deavensoft/paketomat/user/UserController.java

```diff
@@ -24,48 +24,72 @@ public class UserController {
 
     private UserService userService;
 
+    private UserServiceImpl userServiceImpl;
```

Comment:
We should depend on abstractions instead of concrete implementations.

- [ ] Remove UserServiceImpl and use UserService.

Reply:
Done